.travis.yml

```diff
@@ -1,47 +1,92 @@
+sudo: false
 
 language: cpp
 
-compiler:
- - clang
-
 env:
- - GOPATH=${TRAVIS_BUILD_DIR}/go CPPFLAGS="-I/usr/src/gmock/include" CXXFLAGS="-Wall -Werror"
+  global:
+    - INSTALL_DIR=${TRAVIS_BUILD_DIR}/../install
+    - PROTOC="${INSTALL_DIR}/bin/protoc"
```

Comment:
This is all right (especially as I like make variables like CC, CXX, LD, and, now, PROTOC), but setting `PATH="${PATH}:${INSTALL_DIR}/bin"` might have worked in a slightly more generic way (if more tools that we build and install were to be needed).

Don't change, it's all right! We've waited enough... :wink: